driver-core/src/main/java/com/datastax/driver/core/querybuilder/Clause.java

```diff
@@ -111,6 +111,78 @@ boolean containsBindMarker() {
         }
     }
 
+    static class ContainsClause extends AbstractClause {
+
+        private final List<Object> values;
+
+        ContainsClause(String name, List<Object> values) {
+            super(name);
+            this.values = values;
+
+            if (values == null)
+                throw new IllegalArgumentException("Missing values for CONTAINS clause");
+            if (values.size() > 1)
+                throw new IllegalArgumentException("Too many values for CONTAINS clause, only one value allowed");
+        }
+
+        @Override
+        void appendTo(StringBuilder sb, List<Object> variables) {
+
+
+            Utils.appendName(name, sb).append(" CONTAINS ").append("'"+values.get(0)+"'");
```

Comment:
shouldn't this use appendValue(values, sb, variables) so values appropriately gets tracked in variables?